File: NegocioCSharp/Basicas/Funciones.cs
using Microsoft.VisualBasic;
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
namespace LiquidacionSueldos
{
    namespace Negocio
    {
        public class Funciones
        {
            static Datos.Gestor ocdGestor = new Datos.Gestor();
            static DataTable Tabla = new DataTable();

            static string Cadena = "";
            public static string CantidadConLetra(string NumeroDecimal)
            {
                Cadena = "";
                try
                {
                    Tabla = ocdGestor.EjecutarReaderSql("select dbo.CantidadConLetra(" + NumeroDecimal.Replace(",", ".") + ") as CantidadEnLetra");
                    Cadena = Tabla.Rows[0]["CantidadEnLetra"].ToString();
                }
                catch (Exception oError)
                {
                    throw oError;
                }

                return Cadena;
            }
        }
    }
}

File: Negocio/Basicas/Funciones.vb
Namespace LiquidacionSueldos
    Namespace Negocio
        Public Class Funciones
            Shared ocdGestor As New LiquidacionSueldos.Datos.Gestor
            Shared Tabla As New DataTable
            Shared Cadena As String = ""

            Public Shared Function CantidadConLetra(ByVal NumeroDecimal As String) As String
                Cadena = ""
                Try
                    Tabla = ocdGestor.EjecutarReaderSql("select dbo.CantidadConLetra(" & NumeroDecimal.Replace(",", ".") & ") as CantidadEnLetra")
                    Cadena = Tabla.Rows(0)("CantidadEnLetra").ToString()
                Catch oError As Exception
                    Throw oError
                End Try

                Return Cadena
            End Function
        End Class
    End Namespace
End Namespace
